### PR TITLE
test: unit test report circle

### PR DESCRIPTION
**Pull request by wawanrandianto1**
add unit test for report [link issue](https://github.com/comimafun/catalog-api/issues/54)

### CODE REVIEW

**Comment by wawanrandianto1** [comment]
tambah bruno juga di PR ini [link bruno](https://github.com/comimafun/catalog-api/issues/53)

**Comment by althafdaa** [comment]
kalau nambah test judul PR nya `test: ...` ya

**Comment by wawanrandianto1** [comment]
> kalau nambah test judul PR nya `test: ...` ya

waduh iya, sorry

File: tests/test_helper/test_helper.go [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+// 		t.Fatalf("Could not start postgres container: %s", err)
+// 	}
+
+// 	connUrl, err := container.ConnectionString(ctx, "sslmode=disable")
+// 	if err != nil {
+// 		t.Fatalf("Could not get connection string: %s", err)
+// 	}
+
+// 	return connUrl, container
+// }
```
Comment (althafdaa) [inline tests/test_helper/test_helper.go]:
```suggestion
func GetConnURL(t *testing.T, ctx context.Context) (string, testcontainers.Container) {
	container, err := postgres.Run(ctx,
		"postgres:16-alpine",
		postgres.WithDatabase("testdb"),
		postgres.WithUsername("user"),
		postgres.WithPassword("foobar"),
		testcontainers.WithWaitStrategy(
			wait.ForLog("database system is ready to accept connections").
				WithOccurrence(2).
				WithStartupTimeout(5*time.Second),
		),
	)

	if err != nil {
		t.Fatalf("Could not start postgres container: %s", err)
	}

	connUrl, err := container.ConnectionString(ctx, "sslmode=disable")
	if err != nil {
		t.Fatalf("Could not get connection string: %s", err)
	}

	return connUrl, container
}
```
pakai ini aja harusnya ga deprecated
https://testcontainers.com/modules/postgresql/

**Review by althafdaa — APPROVED** [review]
lgtm

**Pull request merged**